formatter.go

```diff
@@ -54,6 +55,11 @@ func (f *Formatter) Format(amount int64) string {
 	return sa
 }
 
+// ToMajorUnits returns float64 representing the value in sub units using the currency data
+func (f *Formatter) ToMajorUnits(amount int64) float64 {
```

Comment:
One small performance thing. If `f.Fraction == 0` then just return `float64(amount)` instead of doing calculations.

Reply:
@danielwitz Hi,
How do you think about this ?
Are there any matters to fix it ?

Reply:
sure will fix

Reply:
@Rhymond ready for another review